CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 * Fix: BeamMP maxplayers that was displaying player count (By @dgibbs64 #551)
 * Fix: BeamMP filter servers by address, not host (By @Rephot #558)
 * Palworld - Replace old and broken protocol with the new one (#560)
+* Updated some wordings around the new Palworld mechanic
```

Comment:
This isn't needed as we have already edited the palworld notes in the reimplementation PR.

Reply:
Removed it.